_posts/angelscript/2021-12-09-chaos_hammer_as_classes.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Classes
```

Comment:
Title should be more declarative;
```suggestion
title: Chaos Hammer AngelScript Classes
```

Reply:
Do this for every sub page, it's necessary because the tab gets the title from this, so people would only see "classes" at the top of the page / in the tab title and might not understand what they were looking at.

Reply:
Resolved in: [`9deb20f` (#231)](https://github.com/momentum-mod/docs/pull/231/commits/9deb20f2fce85754f9071804672499be96686d72)